File: js/programmers/level.0/prgm_120907_1.js
function solution(quiz) {
  let array = [];
  let result = [];
  for (i = 0; i < quiz.length; i++) {
    array.push(quiz[i].split(" "));
  }
  for (j = 0; j < array.length; j++) {
    if (array[j][1] === "+") {
      Number(array[j][0]) + Number(array[j][2]) === Number(array[j][4])
        ? result.push("O")
        : result.push("X");
    } else {
      Number(array[j][0]) - Number(array[j][2]) === Number(array[j][4])
        ? result.push("O")
        : result.push("X");
    }
  }
  return result;
}

File: python/programmers/level.0/prgm_120907_1.py
def solution(quiz):
    result = []
    for q in quiz:
        x, operator, y, equal, z = q.split(" ")
        sum = 0

        if operator == "+":
            sum = int(x) + int(y)
        else:
            sum = int(x) - int(y)
        
        result.append("O") if sum == int(z) else result.append("X")

    return result